### DISCUSSION TITLE
about the 2nd order force constants in quantum espresso format

### DISCUSSION THREAD

**Discussion by ybC007** [ttadano/alamode#95] [General]
Dear Dr. Tadano,
Hello, 
I found that alm provides the 2nd order force constant in the quantum format, but the tag is not shown in the manual. I try to add
the `FC2_QEFC` tag  in the input file to get the `.fc` file. The format of the `.fc` file seems to be different with the result generated by `q2r.x`. 
This maybe a deprecated feature? Or it's just a bug?
Thank you very much in advance!
Looking forward to your kind advice.

**Comment by ttadano** [comment]
`FC2_QEFC` is still not supported. Only tags listed https://alamode.readthedocs.io/en/latest/almdir/inputalm.html#list-of-supported-input-variables are supported.

**Comment by ybC007** [comment]
Okay, got it. 
Thank you for your reply.
